gp-templates/translation-row.php

```diff
@@ -62,7 +66,7 @@
 		<a href="#" row="<?php echo $t->row_id; ?>" class="action edit"><?php _e( 'Details', 'glotpress' ); ?></a>
 	</td>
 </tr>
-<tr class="editor <?php echo $warning_class; ?>" id="editor-<?php echo $t->row_id; ?>" row="<?php echo $t->row_id; ?>">
+<tr class="editor <?php echo esc_attr( gp_get_translation_row_warning_class( $t ) ); ?>" id="editor-<?php echo esc_attr( $t->row_id ); ?>" row="<?php echo esc_attr( $t->row_id ); ?>">
```

Comment:
Now I see why the extra function exists. Is there any harm in using `gp_translation_row_classes()` here too?